pkg/api-server/testdata/versions.json

```diff
@@ -39,6 +39,9 @@
   "~1.4.0": {
    "envoy": "~1.18.4"
   },
+  "~1.4.0-1-anyprerelease": {
+   "envoy": "~1.18.4"
+  },
```

Comment:
How come do we need this?

Reply:
If I get it right, because we didn't push a new tag, it fails on `release-1.5` branch https://app.circleci.com/pipelines/github/kumahq/kuma/12779/workflows/0abb10c2-0d12-444e-ac8e-2452caebe875/jobs/136056

Reply:
Right but this should be 1.5.0-anyprerelease right? Feels like the minor number is not right here

Reply:
Jakub already added `~1.5.0-1-anyprerelease`, but he replaced `~1.4.0-1-anyprerelease` instead of append. We need both for now